channels-rke2.yaml

```diff
@@ -1679,4 +1679,23 @@ releases:
         repo: rancher-rke2-charts
         version: 1.14.400
     featureVersions: *featureVersions-v1
-
+  - version: v1.28.5+rke2r1
+    minChannelServerVersion: v2.8.0-alpha1
+    maxChannelServerVersion: v2.8.99
+    serverArgs: *serverArgs-v1-27-7-rke2r2
+    agentArgs: *agentArgs-v1-25-15-rke2r2
+    charts: &charts-v1-28-5-rke2r1
```

Comment:
You also need to update metrics-server: https://github.com/rancher/rke2/blob/adcd936ed6d7280080fba3670208cdea532de453/charts/chart_versions.yaml#L20

Reply:
Updated